### PR TITLE
Building Desktop version

### PR DESCRIPTION
**Pull request by Caren-Koroeny**
- Created a `desktop` version of the projecet.
- I used some `media queries`.
- Use more css stying that before.
- Implemented all the colors as per `figma` requirements .
- Used `flex box` and `g`rid where necessary.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -8,656 +8,1472 @@
...
+  }
+
+  .contanct-containter {
+    display: flex;
+    margin-top: 60%;
+    max-width: 365px;
+    margin-left: -5%;
+    background-color: #6070ff;
+    border-top-left-radius: 100px;
+  }
```
Comment (binhussen) [inline css/style.css]: - Kindly, be sure your mobile version design still sticks to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=xmWXLXW7NiSVxIum-0), you lose your mobile style when you style the desktop. 👍👌


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

File: css/style.css [hunk 2]
```diff
@@ -8,656 +8,1472 @@
...
-  width: 360px;
-  height: 620px;
-  background: #fff;
-  border: 1px solid #dfe1e6;
-  border-radius: 16px;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  margin: 10px;
-}
```
Comment (binhussen) [inline css/style.css:204]: - Kindly, be sure your mobile version still sticks to the [Figma design](https://user-images.githubusercontent.com/52010789/206548470-985b609a-a380-452f-8a6c-26578dff7c6d.png) 👌👌 here your technology list overlaps on the description, please fix it 👀😱



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

File: css/style.css [hunk 3]
```diff
@@ -8,656 +8,1472 @@
...
+    width: 100%;
+    height: 100%;
+    overflow-x: hidden;
+    padding: 0;
+    margin: 0;
+  }
+
+  .logo-part {
+    display: none;
+  }
```
Comment (binhussen) [inline css/style.css]: - Kindly, be sure that your headline part sticks to the [Figma design](figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=xmWXLXW7NiSVxIum-0), you missed the background image and header padding, please fix it👌👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

File: index.html [hunk 4]
```diff
@@ -284,37 +351,34 @@ <h3 class="language">CSS</h3>
...
         <div class="backgound-image"></div>
-
+        <div class="backgound-image desktop">
           <div class="contact-title">
-            <h2 class="title2">Contact me</h2>
-          </div>
-          <div class="contacts-info">
-            <p>
+            <h2 class="title2 desktop">Contact me</h2>
+            <p class="contacts-info">
```
Comment (binhussen) [inline index.html]: - Kindly, make your contact form like the [Figma design](figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=xmWXLXW7NiSVxIum-0), you added unnecessary margin and the form is not centered 👌👌


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

File: index.html [hunk 5]
```diff
@@ -183,36 +235,50 @@ <h1 class="project-title">Tonic</h1>
     <div class="card">
       <img src="./images/image-4.png" alt="" class="image">
       <div class="left-card1-block">
-        <h1 class="project-title">Multi-Post <br> Stories</h1>
+        <h1 class="project-title mobile">Multi-Post <br> Stories</h1>
+        <h1 class="project-title desktop">Uber <br> Navigation</h1>
```
Comment (binhussen) [inline index.html]: - Kindly, your card image is not given the padding as the Figma, please stick to the Figma as match as possible.👌👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Caren-Koroeny , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- kindly, stick to the [Figma design](figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=xmWXLXW7NiSVxIum-0) as much as possible.👌
- Kindly, be sure your new style doesn’t affect the mobile version.👌👌

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/desktop-styling.css [hunk 6]
```diff
@@ -0,0 +1,771 @@
...
+    padding: 0 75%;
+  }
+
+  .contanct-containter {
+    margin-top: 10%;
+    display: flex;
+    background-color: #6070ff;
+    border-top-left-radius: 100px;
+    position: relative;
+  }
```
Comment (Qoosim) [inline css/desktop-styling.css:625]: - [ ] As suggested by the previous reviewer, you should stick to the figma design as close as possible. Kindly consider checking the width of the container of both the text and the form on the figma design and use the width on your design so as to produce a close replica of the guiding template. Take look at both designs and walk your design alongside the figma design.

### Your Design Desktop
your-form

### Figma Design Desktop
figma-form

File: css/desktop-styling.css [hunk 7]
```diff
@@ -0,0 +1,771 @@
...
+    left: 0;
+    right: 0;
+    top: 0;
+    height: 540px;
+    bottom: 0;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 24px;
+    margin-top: 10%;
+  }
```
Comment (Qoosim) [inline css/desktop-styling.css:178]: - [ ] As suggested by the previous reviewer, you must stick to the figma design as close as possible. You're not very consistent about the way your styled the card for each project under the projects section. Kindly check the two designs screenshots and ensure you give the same margin or padding to the both side of the project box and make it the same for whole cards.
### Your Design
your-card-box
your-card-box-2

### Figma Design
figma-card-box-1
figma-card-box-2

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Caren-Koroeny, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Descriptive PR title and summary. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/desktop-styling.css (hunk 6 above)
Comment (Caren-Koroeny) [reply css/desktop-styling.css:585]: Hello @Qoosim thank you so much for the feedback, okay kindly what am I supposed to do, reduce the size?? Sincein my machine it's showing the same results as figma...kindly what exactly am I supposed to do here
Screenshot from 2022-12-09 13-06-03

Comment (Caren-Koroeny) [reply css/desktop-styling.css:585]: Screenshot from 2022-12-09 13-07-58

Comment (Caren-Koroeny) [reply css/desktop-styling.css:625]: - I will change the border-radius,more details on what else I should change will be appreciated. Thank you already.

File: css/desktop-styling.css (hunk 7 above)
Comment (Caren-Koroeny) [reply css/desktop-styling.css:178]: - Sure let me work on it ASAP, thank you for the clarity.

Comment (Debas-31) [reply css/desktop-styling.css:178]: - [ ] Well done on implementing the suggestion by the previous code reviewer, but still, you need to figure out the alignment between the screenshot and the project details not appropriately aligned. I suggest giving some percentage value to the container width for both the image and project detail then you give padding and margin to align them. :+1: 

***Your design in desktop version***
>Screenshot from 2022-12-09 17-59-19

**Review by Debas-31 — CHANGES_REQUESTED** [review]
> ## Hi @Caren-Koroeny  :wave:, 
# Status: Require Change :recycle:
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- GitHub flow :heavy_check_mark: 
- PR title and summary :heavy_check_mark: 
- Commit message :heavy_check_mark:
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @Caren-Koroeny 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Nice commit history 💯 
✅ Nice usage of HTML tags 💯 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**